documentation/azure-powershell-release-template.md

```diff
@@ -0,0 +1,88 @@
+    - If this PSRepository does not show up, run the following command to register it: `Register-PSRepository -Name dtgallery -SourceLocation http://dtlgalleryint.cloudapp.net/api/v2 -PublishLocation http://dtlgalleryint.cloudapp.net/api/v2 -InstallationPolicy Trusted`
+- [ ] On the evening of MM/DD/YYYY, publish the latest RC of Azure PowerShell to internal through the **ps-pub-test** Jenkins job and smoke test from [here](dtlgalleryint.cloudapp.net/api/v2)
+    - Install the modules for testing by running the following commands: `Install-Module -Repository dtlgallery -Name AzureRM` and `Install-Module -Repository dtlgallery -Name Azure -AllowClobber`
+    - Run the **RunInstallationTests.ps1** script again (same one from the test signoff document) and ensure that all tests passed
+- [ ] Install [Azure Storage Explorer](https://azurestorageexplorer.codeplex.com/) and upload the .msi file to the download blob container
+    - The storage account name and storage account key can be retrieved from the OneNote page
+    - Upload the **azure-powershell.X.X.X.msi** file from the shared folder to the latest downloads folder
+- [ ] Submit a pull request to the WebPI feed and send an email to Chris Sfanos asking to merge the PR
+    - The pull request should just include changes to the **WebProductList_AzurePS.xml** and **DH_AzurePS.xml** files
+- [ ] Update aka.ms link to point at the newest .msi file for Azure PowerShell
+    - Request access if you don't have access to **azure-powershellget**
+- [ ] Enable the live publishing job, **ps-publish**
+- [ ] Run the job against the release branch, which will publish to the PowerShell Gallery
+- [ ] Once the job is complete, disable the **ps-publish** job
+- [ ] Follow the same steps for smoke testing that we followed above for **ps-pub-test**
+- [ ] Update the README and change log in the release branch if necessary
+- [ ] Create a public GitHub [Release](https://github.com/Azure/azure-powershell/releases) with appropriate tag
+    - Use the change log from this release as the content of the document
+    - Include the .msi file and include a link to it at the top of the document with "Installer: Windows Standalone"
+    - Include the changes since the last release at the end of the document
+- [ ] Merge the release branch to master
+- [ ] Delete the release branch
+- [ ] Merge the master branch to dev
+- [ ] Smoke test the live WebPI feed
+- [ ] Send an email to [all partners](mailto:adxsdkpartners@microsoft.com) with the locations of where the release can be accessed: the PowerShell Gallery modules (both Azure and AzureRM), the WebPI feed, and the GitHub release
+    - Copy the e-mail contents and post it on the slack channels (specifically [#powershell](https://azuresdk.slack.com/messages/powershell/) and [#powershell-announce](https://azuresdk.slack.com/messages/powershell-announce/))
+- [ ] Resolve any issues, close the release milestone, and create a new milestone for +3 months
```

Comment:
Contact partners about any of their bugs still open in the release, and ensure bugs are closed or moved into the appropriate milestone